static/bidder-info/deepintent.yaml

```diff
@@ -1,5 +1,5 @@
 maintainer:
-  email: "sourabh@deepintent.com"
+  email: "prebiddev@deepintent.com"
```

Comment:
Not sure how we missed that one. Thank you for changing this to a mailing group instead of an individual contact.

Reply:
No Problem 😄